tests/trainer/test_trainer.py

```diff
@@ -2398,7 +2398,7 @@ def test_bnb_adam8bit_no_bnb(self):
 
         # Pretend that bnb does not exist, even if installed. By setting bnb to None, importing
         # bnb will fail even if bnb is installed.
-        with patch.dict("sys.modules", {"bnb.optim": None}):
+        with patch.dict("sys.modules", {"bitsandbytes.optim": None}):
```

Comment:
So now `bitsandbytes` is invisible with this change?

Reply:
Yes inside this context manager bistandbytes should be invisible. Before it tried to overwrite the bnb but it didn't existed (you can also check the lines above)

Reply:
I personally wasn't even sure why we needed the mock test when we have the actual library to test with. But since the effort was made to write it, I said why not.